test/extended/testdata/router/router-http2.yaml

```diff
@@ -0,0 +1,234 @@
+apiVersion: v1
+kind: Template
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: http2
+    annotations:
+      service.beta.openshift.io/serving-cert-secret-name: serving-cert-http2
+  spec:
+    selector:
+      name: http2
+    ports:
+      - name: https
+        protocol: TCP
+        port: 27443
+        targetPort: 8443
+      - name: http
+        protocol: TCP
+        port: 8080
+        targetPort: 8080
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: src-config
+  data:
+    data.base64: |
```

Comment:
I was being consistent with what was done in the gRPC interop testing.

Reply:
> What's the advantage of using a base64-encoded compressed tarball instead of directly putting the files in the configmap?

I didn't need to worry about quoting in any way was one reason and I shared the code I already did in grpc interop tests.

Reply:
> What's the advantage of using a base64-encoded compressed tarball instead of directly putting the files in the configmap?

I think the long term solution here is a separate image for h2spec, grpc-interop and these tests.